src/app/item-card/item-card.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, OnInit, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-item-card',
+  templateUrl: './item-card.component.html',
+  styleUrls: ['./item-card.component.css']
+})
+export class ItemCardComponent implements OnInit {
+
+  @Input() itemName: string;
+  @Input() itemAbout: string;
+  @Input() itemRating: number;
+  @Input() itemPrice: number;
+  @Input() itemImageSrc: string;
+
+  constructor() {
```

Comment:
Since we are not using the constructor or the OnInit interface, we can remove those two empty methods, and the OnInit import.